apps/backend/src/modules/users/user.entity.ts

```diff
@@ -103,12 +123,25 @@ class UserEntity implements Entity {
 	public toObject(): {
 		email: string;
 		firstName: string;
+		group: null | {
+			id: number;
+			key: string;
+			name: string;
+			permissions: {
+				id: number;
+				key: string;
+				name: string;
+			}[];
+		};
+		groupId: null | number;
```

Comment:
groupId can't be null, as well as the group itself; go through all places where you mention group and remove unnecessary null checks if any
```suggestion
		groupId: number;
```

Reply:
I forgot about this, in this case let's leave | null